### ISSUE TITLE
Touch input stopped working properly inside games in game mode

### ISSUE THREAD

**Issue by AlexFolland** [ValveSoftware/SteamOS#1477]
#### Your system information

* Operating System: SteamOS 3.6 Build: 20240417.1000
* KDE Plasma Version: 5.27.10
* KDE Frameworks Version: 5.115.0
* Qt Version: 5.15.12
* Kernel Version: 6.1.52-valve19-1-neptune-61 (64-bit)
* Graphics Platform: X11
* Processors: 8 × AMD Custom APU 0932
* Memory: 14.5 GiB of RAM
* Graphics Processor: AMD Radeon Graphics
* Manufacturer: Valve
* Product Name: Galileo
* System Version: 1
* Opted into Steam client beta?: Yes
* Opted into SteamOS beta?: Yes
* Have you checked for updates in Settings > System?: Yes

#### Please describe your issue in as much detail as possible:

As of today, in game mode specifically, the touch screen no longer clicks in the position where I touched the screen.  Instead, it clicks where the cursor currently is.  If I long-press the screen, the cursor can start following my finger, but just simply touching somewhere on the screen does not click there unless I hold an extremely long time and click again after the cursor has moved.

This happens on both my Steam Deck OLED and my Steam Deck; completely different Steam Decks.

I expect touching something on the screen to click on that thing, like it did before today.

This is an extremely major regression, basically breaking all gameplay for my toddler's Steam Deck while we're on vacation.  Please fix it very soon.

#### Steps for reproducing this issue:

1. Start a mouse-clicking game like Buzzy Explores The Airport or Fatty Bear's Birthday Surprise.
2. Wait through the intro dialog or press the start button until it's done.
3. Tap the touch screen on screen elements you want to interact with.

**Comment by AlexFolland** [comment]
Update: I confirmed that this is a regression in the latest version.  Here's how.

1. I switched to "OS Update Channel" "Stable" and "Steam Client Update Channel" "Steam Deck Stable".
2. I rebooted the Steam Deck and awaited all updates, including all game updates (and literally every installed game got an update).
3. I rebooted again, and ran a game (in this case, Putt-Putt Joins The Parade, one of several games where I saw the issue).
4. I tapped various places on the screen.

The issue was resolved, so this issue seems to only be present in Steam Beta client.
